tests/python/gaia-ui-tests/gaiatest/apps/system/regions/search_panel.py

```diff
@@ -0,0 +1,109 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Following up from the discussion on IRC: we can delete the other rocket bar page class, for now